CHANGELOG.rst

```diff
@@ -7,6 +7,74 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 0.15.0 (2022-08-05)
+---------------------------
+
+* Added: ``audformat.MiscTable``
+  which can store data
+  not associated with media files
+* Added: store scheme labels in a misc table
+* Added: dictionary ``audformat.Database.misc_tables``
+  holding misc tables of a database
+* Added: ``audformat.utils.difference()``
+  for finding index entries
+  that are only part of a single index
+  for a given sequence of indices
+* Added: ``audformat.utils.is_index_alike()``
+  for checking if a sequence of indices
+  has the same number of levels,
+  level names,
+  and matching dtypes
+* Added: ``audformat.define.DataType.OBJECT``
+* Added: ``audformat.utils.set_index_dtypes()``
+  to change dtypes of an index
+* Added: ``audformat.testing.add_misc_table()``
+* Changed: ``audformat.Database.update()``
+  can now join schemes
+  with different labels
+* Changed: ``audformat.utils.union()``
+  and ``audformat.utils.intersect()``
+  now support any kind of index
+* Changed: ``audformat.utils.concat()``
+  can now concatenate objects
+  that contain any kind of index
+* Changed: ``audformat.Database.__iter__``
+  iterates through all (misc) tables,
+  e.g. a user can do ``list(db)``
+* Changed: require ``pandas>=1.4.1``
```

Comment:
Should we mention that this also requires `Python >= 3.7`?

Reply:
It requires `Python >= 3.8`.
So far I decided to just mention at the end, that we removed support for Python 3.7 without any reason